### ISSUE TITLE
Very lengthy loop calculations while "Setting minimum Insulin Peak Time"

### ISSUE THREAD

**Issue by mountrcg** [ivalkou/freeaps#108]
I mistakenly had  the following configuration:
`curve=rapid-acting` 
`usecustompeaktime=true`  
`insulinPeakTime=45`
The later is below the minimum insulinPeakTime of Orefs rapid-acting curve.
FAX raises the following:
`2021-11-02 19:44:15.900073+0100 FreeAPS[826:128377] [OpenAPS] JavaScriptWorker.swift - createContext() - 23 DEV: JavaScript log: Setting minimum Insulin Peak Time of 50m for,rapid-acting,insulin` every 5 ms from
`2021-11-02 19:44:03.263990+0100` until `2021-11-02 19:44:15.979159+0100` delaying every loop calculation considerably

**Comment by mountrcg** [comment]
issue also raised in Telegramm group and analysed by Marion: https://t.me/freeapsx_eng/9783

**Comment by zubernuker** [comment]
Try ultra-rapid insulin curve, it has minimum of 35 min peak time.

https://github.com/openaps/oref0/blob/master/lib/iob/calculate.js#L90

**Comment by mountrcg** [comment]
Yes I did this. However if someone sets a minumum peak time of 30min with ultra-rapid, that person will run in the same problem.
